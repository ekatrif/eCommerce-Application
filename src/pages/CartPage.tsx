import React from 'react';
import Container from '../components/UI/Container/Container';
import H1 from '../components/UI/titles/H1/H1';

const CartPage: React.FC = () => {
  return (
    <div className="content">
      <Container>
        <div className="main">
          <H1 text="Your Cart" />
        </div>
      </Container>
    </div>
  );
};

export default CartPage;
