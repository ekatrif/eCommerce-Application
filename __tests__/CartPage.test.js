import React from 'react';
import renderer from 'react-test-renderer';
import CartPage from '../src/pages/CartPage';

test('CartPage renders correctly', () => {
  const tree = renderer
    .create(<CartPage />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
