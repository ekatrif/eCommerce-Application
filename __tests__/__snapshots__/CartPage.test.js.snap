// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartPage renders correctly 1`] = `
<div
  className="content"
>
  <div
    className="container"
  >
    <div
      className="main"
    >
      <h1
        className="h1"
      >
        Your Cart
      </h1>
    </div>
  </div>
</div>
`;
